utils/gitlab_api.py

```diff
@@ -293,22 +297,42 @@ def get_access_level(access):
         elif access == "guest":
             return gitlab.GUEST_ACCESS
 
+    def get_group_id_and_projects(self, group_name):
+        groups = self.gl.groups.list()
+        group = [g for g in groups if g.path == group_name]
+        if len(group) != 1:
```

Comment:
Then `if not len(group):` should be enough.

Reply:
see previous comment